sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -988,19 +988,13 @@ public partial class DataPointAnomaly
         public static bool operator !=(Azure.AI.MetricsAdvisor.Models.DetectionConditionsOperator left, Azure.AI.MetricsAdvisor.Models.DetectionConditionsOperator right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class DimensionKey : System.IEquatable<Azure.AI.MetricsAdvisor.Models.DimensionKey>
+    public partial class DimensionKey : System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>>, System.Collections.IEnumerable
```

Comment:
We don't have strong user scenarios in which we think this `IEquatable` will be helpful. After discussing with architects, decided to remove it. We can consider adding it in the future if needed.

Just in case, we're making `DimensionKey` immutable. Having mutable `IEquatables` can lead to undesired behavior if they're used as dictionary keys.